packages/dart_frog_cli/pubspec.yaml

```diff
@@ -14,12 +14,13 @@ dependencies:
   cli_completion: ^0.3.0
   dart_frog_gen: ^0.4.0
   equatable: ^2.0.5
-  mason: ^0.1.0-dev.39
+  mason: 0.1.0-dev.50
```

Comment:
Pinning this dependency because Mason may introduce breaking changes on those dev releases and may breaking released versions of the DF cli